content/applications/inventory_and_mrp/inventory/management/warehouses/resupply_warehouses.rst

```diff
@@ -0,0 +1,55 @@
+===============================
+Resupply from another Warehouse
+===============================
+
+A common use case for multiple warehouses is to have one central warehouse that resupplies shops,
+each shop is considered as a local warehouse. When a shop want to replenish a product, this product
+is ordered to the central stock. Odoo allows you to set easily which warehouse(s) can resupply a
+warehouse.
+
+Configuration
+=============
+
+In order to be able to resupply from another warehouse, you need to activate *multi-step routes*.
+
+.. image:: resupply_warehouses/virtual-warehouses-settings.png
+   :align: center
+   :alt: Enable Multi-Step Routes in Odoo settings
+
+You can then access your warehouses via :menuselection:`Inventory --> Configuration -->
+Warehouses`.
+
+Open the warehouse which should be resupplied by the another one. You will have the possibility to
+directly indicate through which warehouse(s) it can be resupplied.
+
+.. image:: resupply_warehouses/resupply-from-second-warehouse.png
+   :align: center
+   :alt: Supply one warehouse with another in Warehouse Configuration tab.
+
+By activating this option, a new route will now be available on your products *Supply Product from
+Second warehouse*. It can now be selected, along with either a *reordering rule* or a *make to
+order*.
+
+.. image:: resupply_warehouses/product-resupply-route-settings.png
+   :align: center
+   :alt: Route setting which enables a product to resupplied from a second warehouse.
+
+In this example, a reordering rule is set with a minimum of 5 units in stock and a maximum of 10
```

Comment:
Replace the four sentences on Lines 37-43 `In this example...a transit location.` with:

``When a product's reordering rule is triggered and the product has the :guilabel:`Supply Product from [Warehouse Name]` route set, Odoo automatically creates two pickings. One picking is a *delivery order* from the second warehouse, which contains all the necessary products, and the second picking is a *receipt* with the same products for the main warehouse. The product move from the second warehouse to the main warehouse is fully tracked in Odoo.``
[line break]
``On the picking/transfer records created by Odoo, the :guilabel:`Source Document` is the product's reordering rule. The location between the delivery order and the receipt is a transit location.``